README.md

```
@@ -1,20 +1,59 @@
+
+    add_filter do |src|
+      !tracked.include?(src.filename)
+    end
+  end
+  ENV['COVERAGE_STARTED'] = 'true'
+end
+```
+
+Then, run the test suite using the following:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)